tests/src/test/scala/whisk/core/cli/test/WskBasicUsageTests.scala

```diff
@@ -148,7 +148,7 @@ class WskBasicUsageTests
         }
     }
 
-    it should "show api build using http apihost" in {
+    it should "show api build using http apihost with protocol specified" in {
```

Comment:
this is the area of code i was considering
```
+    if !strings.HasPrefix(host, "http") {
+        host = "https://" + host
+    }
```
so i was thinking an apihost setting with and without a http prefix should work.  will a protocol-less apihost fail?